packages/enzyme/src/ReactWrapper.js

```diff
@@ -1095,7 +1095,7 @@ class ReactWrapper {
     const fnName = typeof name === 'string' ? name : 'unknown';
     const callback = typeof fn === 'function' ? fn : name;
     if (this.length !== 1) {
-      throw new Error(`Method “${fnName}” is only meant to be run on a single node. ${this.length} found instead.`);
+      throw new Error(`Method “${fnName}” is meant to be run on 1 node. ${this.length} found instead.`);
```

Comment:
Can you elaborate on why this change is clearer to you? "single" means "1".

Reply:
When I scan the message, I pick up faster on the number 1 than the words "a single".  Because `this.length` is an integer, I find it easier to compare the two integers.